x/gov/types/params.go

```diff
@@ -152,7 +165,16 @@ func validateTallyParams(i interface{}) error {
 		return fmt.Errorf("vote threshold must be positive: %s", v.Threshold)
 	}
 	if v.Threshold.GT(sdk.OneDec()) {
-		return fmt.Errorf("vote threshold too large: %s", v)
+		return fmt.Errorf("vote threshold too large: %s", v.Threshold)
+	}
+	if !v.ExpeditedThreshold.IsPositive() {
+		return fmt.Errorf("expedited ote threshold must be positive: %s", v.ExpeditedThreshold)
+	}
+	if v.ExpeditedThreshold.GT(sdk.OneDec()) {
+		return fmt.Errorf("expedited vote threshold too large: %s", v.ExpeditedThreshold)
+	}
+	if v.ExpeditedThreshold.LT(v.Threshold) {
```

Comment:
```suggestion
	if v.ExpeditedThreshold.LTE(v.Threshold) {
```

Reply:
ditto: I will update locally because err messages need to be updated as well